tests/SearchTestCase.php

```diff
@@ -229,7 +229,7 @@ public function testUtf8AnalyzerKnowsToHandleOtherEncodingsWorks()
         $analyzer = new Doctrine_Search_Analyzer_Utf8();
 
         // convert our test string to iso8859-15
-        $iso = iconv('UTF-8','ISO8859-15', 'un éléphant ça trompe énormément');
+        $iso = mb_convert_encoding('un éléphant ça trompe énormément', 'ISO-8859-15', 'UTF-8');
```

Comment:
You could also stick to iconv, as it will be introduced as an external requirement with this change.

Reply:
Merge as-is, I have the feeling that using the multi-byte string extension is the cleanest solution.
Unfortunately we can not adapt the other usages for now